### ISSUE TITLE
Text-Align in Right to left langs

### ISSUE THREAD

**Issue by manisi** [jhipster/generator-jhipster#6309]
##### **Overview of the issue**

 Text align is not set to right for right to left languages. It should be set to right, especially for label tags.

##### **Motivation for or Use Case**

when set languages to farsi (RTL) text-align not set to right.

##### **JHipster configuration**

Using JHipster version installed locally in current project's node_modules
Executing jhipster:info
Options: 
Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**

```
sample-47@0.0.0 /Users/developer/Desktop/sample47
└── generator-jhipster@4.7.0 

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.mycompany.myapp",
      "nativeLanguage": "en"
    },
    "jhipsterVersion": "4.7.0",
    "baseName": "sample47",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "hibernateCache": "ehcache",
    "clusteredHttpSession": false,
    "websocket": "spring-websocket",
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "mysql",
    "searchEngine": "elasticsearch",
    "messageBroker": false,
    "serviceDiscoveryType": "eureka",
    "buildTool": "maven",
    "enableSocialSignIn": false,
    "enableSwaggerCodegen": true,
    "jwtSecretKey": "replaced-by-jhipster-info",
    "clientFramework": "angularX",
    "useSass": true,
    "clientPackageManager": "yarn",
    "applicationType": "monolith",
    "testFrameworks": [
      "gatling"
    ],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": [
      "en",
      "fa"
    ]
  }
}
</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>
entity Foo (foo) {
  name String,
  salary BigDecimal,
  date LocalDate required
}

dto Foo with mapstruct
paginate Foo with pagination
service Foo with serviceImpl

</pre>
</details>

##### **Environment and Tools**

java version "1.8.0_60"
Java(TM) SE Runtime Environment (build 1.8.0_60-b27)
Java HotSpot(TM) 64-Bit Server VM (build 25.60-b23, mixed mode)

git version 2.10.1 (Apple Git-78)

node: v8.2.1

npm: 5.3.0

bower: 1.8.0

yarn: 0.27.5

Docker version 17.05.0-ce, build 89658be

docker-compose version 1.13.0, build 1719ceb

Execution complete

##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>
entity Foo (foo) {
  name String,
  salary BigDecimal,
  date LocalDate required
}

dto Foo with mapstruct
paginate Foo with pagination
service Foo with serviceImpl

</pre>
</details>

##### **Browsers and Operating System**

Mac OS X 10.11.6   - All Browser

- [ ] Checking this box is mandatory (this is just to show you read everything)

**Comment by jdubois** [comment]
Should be fixed by #6317 (but I can't test it, so please re-open if this isn't fixed in the next release)

**Comment by manisi** [comment]
Thanks  @jdubois  !
I will certainly test this item in future versions.
